### DISCUSSION TITLE
Error : cloud not find a production build in the directory

### DISCUSSION THREAD

**Discussion by JSDUNIYA** [vercel/next.js#48734] [Help]
### Summary

I am running next.renderTOHTML(), 

next.renderTOHTML(req, headers, req.path); 

I am getting above error "cloud not find a production build in the directory" ? any one ran into same error?

**Comment by Hamzi-SE** [comment]
Check that the .next directory exists in the root directory of your application.

If the .next directory is missing, try running the `next build` command again to build your application.

**Reply by JSDUNIYA** [reply]
it is creating but single-spa gives me this error while running the app.

**Reply by pratiks-wellsky** [reply]
same error
